unison-share-api/src/Unison/Server/Endpoints/NamespaceListing.hs

```diff
@@ -185,52 +178,9 @@ serve codebase maySCH mayRelativeTo mayNamespaceName = do
   let namespacePath = fromMaybe Path.empty mayNamespaceName
   let path = relativeToPath <> namespacePath
   let path' = Path.toPath' path
-
-  case (useIndex, mayRootHash) of
-    (True, Nothing) ->
-      serveFromIndex codebase mayRootHash path'
-    (True, Just rh)
-      | rh == codebaseRootHash ->
-          serveFromIndex codebase mayRootHash path'
-      | otherwise -> do
-          serveFromBranch codebase path' rh
-    (False, Just rh) -> do
-      serveFromBranch codebase path' rh
-    (False, Nothing) -> do
-      ch <- liftIO $ Codebase.runTransaction codebase Operations.expectRootCausalHash
-      serveFromBranch codebase path' ch
-
-serveFromBranch ::
```

Comment:
It turns out serving from Index or not is actually the same for this particular endpoint; not sure when that changed, but it's nice to just merge them together 🙌🏼